examples/spring-otel/src/main/java/com/commercetools/sdk/examples/springmvc/service/CtpClientBeanService.java

```diff
@@ -30,16 +30,22 @@ public class CtpClientBeanService {
     @Value(value = "${ctp.project.key}")
     private String projectKey;
 
+    @Value(value = "${otel.provider:local}")
+    private OtelProvider otelProvider;
+
     private ClientCredentials credentials() {
         return ClientCredentials.of().withClientId(clientId).withClientSecret(clientSecret).build();
     }
 
     @Bean
     public ProjectScopedApiRoot apiRoot() {
-        return ApiRootBuilder.of()
+        ApiRootBuilder builder = ApiRootBuilder.of()
                 .defaultClient(credentials())
-                .withSerializer(new OpenTelemetryResponseSerializer(ResponseSerializer.of(), GlobalOpenTelemetry.get()))
-                .withTelemetryMiddleware(new OpenTelemetryMiddleware(GlobalOpenTelemetry.get()))
-                .build(projectKey);
+                .withTelemetryMiddleware(new OpenTelemetryMiddleware(GlobalOpenTelemetry.get(),
+                        otelProvider.supportsHistogram()));
+        if (otelProvider.supportsHistogram()) {
```

Comment:
maybe add a second flag  `useOtelSerializer` to the Enum so we are more explicit in the configuration

Reply:
https://github.com/commercetools/commercetools-sdk-java-v2/pull/524/commits/cfe8e23673a66dfb574508786ba922a9ef32df01